pkgs/shelf_web_socket/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.0.1-wip
+
+* Replace `dependency_overrides` on `package:test` `1.25.2` with a
```

Comment:
Drop this description. Doesn't help end users. Leave the place-holder release version here, though

Reply:
Done.